mllib/src/main/scala/org/apache/spark/mllib/linalg/Matrices.scala

```diff
@@ -77,8 +77,8 @@ sealed trait Matrix extends Serializable {
     C
   }
 
-  /** Convenience method for `Matrix`-`DenseVector` multiplication. */
-  def multiply(y: DenseVector): DenseVector = {
+  /** Convenience method for `Matrix`-`Vector` multiplication. */
+  def multiply(y: Vector): DenseVector = {
```

Comment:
We cannot delete `def multiply(y: DenseVector)`, which breaks binary compatibility. Please delegate the implementation to `multiply(y: Vector)` and update Mima excludes.